### PR TITLE
docs: add release process for website and docs

### PR DESCRIPTION
**Pull request by tomleah**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [ ] website builds and runs in production - for information on how to test locally go [here](../DEVELOPMENT.md).

## Description
Adds the release process for deploying the website with the latest documentation

## Related Issues
fixes: https://github.com/appsody/website/issues/54

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+git clone https://github.com/appsody/website.git
+cd website
+```
+
+2. Check remote in `package.json`
+```
+"deploy": "gatsby clean && gatsby build --prefix-paths && gh-pages -d public -o <remote>"
+```
+The remote needs to be set to the upstream project, but my default points to a `staging` remote.
```
Comment (mhamwala) [inline RELEASE.md]: `but my` -> `but the`

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+3. Deploy website manually
+```
+npm run deploy
+```
+
+## Release schedule
+The website and documentation is released after every pull request merge. The website is currently **not** versioned and their for an offical release is not cut.
```
Comment (mhamwala) [inline RELEASE.md]: `is released..` -> `gets released when a pull request is merged`

`and thier for` -> `and therefore`

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Appsody Website and Documentation Release Process
+This document outlines the release process for both the website and the documentation.
+
+Both the website and the documentation are combined and theirfore will be released at the same time when either of the projects are modified.
```
Comment (mhamwala) [inline RELEASE.md]: we could reword this sentence to something like:?

>Documentation is stored within the website repository, the website will be redeployed when a change is made to either the website itself or the documentation.

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Appsody Website and Documentation Release Process
```
Comment (mhamwala) [inline RELEASE.md]: perhaps just:?

>Release Process for Website and Documentation

Comment (mhamwala) [reply RELEASE.md]: prefer later

File: RELEASE.md (hunk 1 above)
Comment (tomleah) [reply RELEASE.md]: I changed `my` -> `the`. I want to avoid "should"

File: RELEASE.md (hunk 3 above)
Comment (tomleah) [reply RELEASE.md]: Thanks 🙂

File: RELEASE.md (hunk 4 above)
Comment (tomleah) [reply RELEASE.md]: Went with the later 🙂

**Pull request merged**